less/home-page.less

```diff
@@ -46,9 +92,23 @@
     display: flex;
     justify-content: center;
 
+    @media @mobile {
+      display: flex;
+      flex-direction: column-reverse;
+      justify-content: center;
+      align-items: flex-start;
+
```

Comment:
inside the media queries is not necessary to repeat properties you already used,`display: flex` is not really necessary since you already  set `display: flex`